pom.xml

```diff
@@ -32,7 +32,9 @@
   </developers>
 
   <properties>
-    <powermock.version>1.5.5</powermock.version>
+    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
With the upgrade to the new test harness, tests start failing with the following error:

> testNoGroupsAreRegistered(hudson.plugins.active_directory.RemoveIrrelevantGroupsTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.NoClassDefFoundError: jenkins/model/DirectlyModifiableTopLevelItemGroup
	at hudson.plugins.active_directory.RemoveIrrelevantGroupsTest.<init>(RemoveIrrelevantGroupsTest.java:56)
Caused by: java.lang.ClassNotFoundException: jenkins.model.DirectlyModifiableTopLevelItemGroup
	at hudson.plugins.active_directory.RemoveIrrelevantGroupsTest.<init>(RemoveIrrelevantGroupsTest.java:56)

Looking at [DirectlyModifiableTopLevelItemGroup](https://github.com/jenkinsci/jenkins/blob/9fce1ee933eb5276baff977d562fc8e183f1c8d6/core/src/main/java/jenkins/model/DirectlyModifiableTopLevelItemGroup.java), you can see that it exists since `1.548` (which is newer than `1.533`). To avoid the error above, we need `1.548+`, I have chosen `1.580.1` because it is an LTS and includes SECURITY-144